frontend/src/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -98,6 +101,26 @@ class Selectbox extends React.PureComponent<Props, State> {
     )
   }
 
+  /**
+   * Both onInputChange and onClose handle the situation where
+   * the user has hit backspace enough times that there's nothing
+   * left in the input, but we don't want the value for the input
+   * to then be invalid once they've clicked away
+   */
+  private onInputChange = (event: any) => {
+    const currentInput = event.target.value
+
+    this.setState({
+      isEmpty: !currentInput,
+    })
+  }
+
+  private onClose = () => {
+    this.setState({
+      isEmpty: false,
+    })
+  }
```

Comment:
I don't love this, but it works. I'd probably refactor this entire component into a functional component in the future, maybe rethink the state here a bit.